ui/templates/listing.html

```diff
@@ -1,83 +1,106 @@
+      <li>
         {{ course.course_number }}
-    </li>
+      </li>
 
-{% endfor %}
-</ul>
+      {% endfor %}
+    </ul>
 
-<h2>Runs</h2>
-<ul>
-{% for run in runs %}
-    <li>
+    <div class="panel panel-default">
+      <div class="panel-heading">
+        <h4 class="panel-title">
+          <a class="accordion-toggle" data-toggle="collapse" data-parent="#accordion" href="#collapse-1">
+            Run
+          </a>
+        </h4>
+      </div>
+    </div>
+
+    <ul>
+      {% for run in runs %}
+      <li>
         {{ run }}
-    </li>
+      </li>
+
+      {% endfor %}
+    </ul>
 
-{% endfor %}
-</ul>
+    <div class="panel panel-default">
+      <div class="panel-heading">
+        <h4 class="panel-title">
+          <a class="accordion-toggle" data-toggle="collapse" data-parent="#accordion" href="#collapse-1">
+            Item Type
+          </a>
+        </h4>
+      </div>
+    </div>
 
-<h2>Resource Type</h2>
-<ul>
-{% for tag in tags %}
-    <li>
+    <ul>
+      {% for tag in tags %}
+      <li>
         {{ tag.name }}
-    </li>
+      </li>
 
-{% endfor %}
-</ul>
+      {% endfor %}
+    </ul>
+  </div>
 </div>
+<div class="col-md-9 col-results">
+  <div id="resources">
 
-<div id="resources">
-<h2>Resource</h2>
-<table>
-<tr><th>Type</th><th>title</th><th>Path</th><th>Export</th></tr>
-{% for resource in resources %}
-    <tr>
+    <table class="table table-striped table-condensed listing-table">
+      <tr><th>Type</th><th>title</th><th>Path</th><th>Export</th></tr>
+      {% for resource in resources %}
+      <tr>
         <td>
-            {{ resource.learning_resource_type.name }}
+          {{ resource.learning_resource_type.name }}
         </td>
         <td>
-            {{ resource.title }}
+          {{ resource.title }}
         </td>
         <td>
-            {{ resource.materialized_path }}
+          {{ resource.materialized_path }}
         </td>
         <td>
-            <a href="{% url 'export' resource.id %}">xml</a>
+          <a href="{% url 'export' resource.id %}">xml</a>
         </td>
-    </tr>
-{% endfor %}
-</table>
-
-{% if resources.has_other_pages %}
-{% if resources.has_previous %}
-<a href="{% url 'listing' repo.slug resources.previous_page_number  %}">page {{ resources.previous_page_number }}</a>
-{% endif %}
-{{ resources }}
-{% if resources.has_next %}
-<a href="{% url 'listing' repo.slug resources.next_page_number  %}">page {{ resources.next_page_number }}</a>
-{% endif %}
-{% endif %}
-</div>
-{% endif %} {# endif for if courses.count > 0, hide all this for empty repos #}
+      </tr>
+      {% endfor %}
+    </table>
+    <div class="row">
+      <div class="col-md-12">
+        <div class="lore-pagination">
+          {% if resources.has_other_pages %}
+          {% if resources.has_previous %}
+          <a href="{% url 'listing' repo.slug resources.previous_page_number  %}"><i class="fa fa-angle-left"></i></a>
+          {% endif %}
+          {{ resources }}
```

Comment:
I thought I could fiddle the css to make the text less obnoxious but it is insidious.  When I changed the font-size of lore-pagination it not only reduced the size of the text (good), but it also shrank the size of the angle-bracket links (not good).  If I hide the text, it changes the way the grid renders for some reason I haven't sussed out.  Harrumph.